lua/lspconfig/prismals.lua

```diff
@@ -0,0 +1,34 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = "prismals"
+local bin_name = "prisma-language-server"
+if vim.fn.has('win32') == 1 then
+  bin_name = bin_name..".cmd"
+end
+
+configs[server_name] = {
+  default_config = {
+    cmd = {bin_name, "--stdio"};
+    filetypes = {"prisma"};
+    settings = {
+      prisma = {
+        prismaFmtBinPath = "";
+      }
+    };
+    root_dir = function(fname)
+      return util.root_pattern(".git")(fname);
```

Comment:
I think this could just be `root_dir = util.root_pattern(".git")`

Reply:
that actually was not working for me